app/services/food_service.rb

```diff
@@ -0,0 +1,15 @@
+class FoodService
+  def find_food(input)
+    conn = Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/") do |faraday|
+      faraday.headers["X-API-KEY"] = "xISxGZSXprkP9YkUCjgzm1nLtO77TgoVQtIvXgwN"
```

Comment:
Careful here! We want to use something to be able to hide our API keys. This is something we do not want to push up to github. To hide away environment variables, you could use something like the Figaro Gem.